src/Moadim/Omer.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Zman\Moadim;
+
+trait Omer
+{
+    /**
+     * The Omer goes from 1 to 49 from Pesach to Shavuos.
+     *
+     * @return bool
+     */
+    public function getOmerCount()
+    {
+        if ($this->jewishMonth == 8 && $this->jewishDay >= 16) {
+            return $this->jewishDay - 15;
+        }
+        if ($this->jewishMonth == 9) {
+            return $this->jewishDay + 15;
+        }
+        if ($this->jewishMonth == 10 && $this->jewishDay <= 5) {
+            return $this->jewishDay + 44;
+        }
+
+        return 0;
+    }

```

Comment:
Can we return `null` when the count is not applicable rather than 0?

Also, what do you think about something like this instead:
```php
$count = $this->diffInDays(
    static::firstDayOfPesach($this->jewishYear)
);

return $count > 0 && $count < 50 ? $count : null;
```

Reply:
I thought about that, but my logic was that the Omer days don't actually change from year to year, and it's slightly quicker and less resource intensive to just check the date than to create another object and compare the difference in days. In the end of the day it's your choice, but that's why I went this route